node/cn/backend.go

```diff
@@ -215,21 +215,13 @@ func New(ctx *node.ServiceContext, config *Config) (*CN, error) {
 	setEngineType(chainConfig)
 
 	// load governance state
+	chainConfig.SetDefaults()
+	// latest values will be applied to chainConfig after NewMixedEngine call
 	governance := governance.NewMixedEngine(chainConfig, chainDB)
+	logger.Info("Initialised chain configuration", "config", chainConfig)
 
-	// Set latest unitPrice/gasPrice
-	chainConfig.UnitPrice = governance.Params().UnitPrice()
```

Comment:
Could you explain or link reason for removing this line for history?

Reply:
@henry-will 
This line was first introduced when we needed to increase the gas fee from 25 ston to 750 ston (March, 2022).
Before then, chainConfig used to mean the initial cypress chainConfig.
However, because of the confusing usage of chainConfig, people that it should contain the latest parameters. (somewhat related to #1615)
This line was to update `chainConfig.UnitPrice` to the latest 750 ston back then.

Now, the same logic is handled by MixedEngine (`HandleUpdateParam()` called from `NewMixedEngine()`). It keeps the chainConfig up to date.
Also, only allowing MixedEngine to update chainConfig would be reasonable because it can only change via vote.